packages/atomic/src/components/search/atomic-layout/atomic-search-layout.pcss

```diff
@@ -52,6 +52,10 @@ atomic-search-layout {
       &::part(suggestions-left) {
         flex-basis: var(--atomic-layout-search-box-left-suggestions-width, 30%);
       }
+
+      &::part(suggestions-right) {
+        flex-basis: calc(100% - var(--atomic-layout-search-box-left-suggestions-width, 30%));
```

Comment:
An alternative to this would've been to set `flex-grow: 0` on the left suggestions. However, the left suggestions do need to grow when there's no right suggestions.